changelogs/2.11.3.md

```diff
@@ -0,0 +1,161 @@
+# 2.11.3
+
+Date: 2024-04-17
+Tag: 2.11.3
+
+## Overview
+
+2.11.3 is the next stable release in the [long-term support (LTS)
+version][release_policy][release_policy] 2.11.x release series.
+
+The label "stable" means there are 1.10.x-based applications running in
+production for quite a while without known crashes, incorrect results or
+other showstopper bugs.
+
+This release introduces 1 improvements and resolves roughly 54
+issues since the 2.11.2 version.
+
+[release_policy]: https://www.tarantool.io/en/doc/latest/release/policy/
+[issues]: https://github.com/tarantool/tarantool/issues
+
+## Compatibility
+
+Tarantool 2.11.x is backward compatible with Tarantool 1.10.x in binary data
+layout, client-server protocol and replication protocol.
+
+Please [upgrade][upgrade] using the `box.schema.upgrade()` procedure to unlock
+all the new features of the 2.11.x series.
+
+[upgrade]: https://www.tarantool.io/en/doc/latest/book/admin/upgrades/
+
+## Functionality added or changed
+
+### Tools
+
+* Added the `--human-readable` option for the `misc.memprof` parser to print
+  sizes like 1KiB, 234MiB, 2GiB, etc.
+  Made the errors from the profilers more user-friendly (gh-9217).
```

Comment:
Let's move it to a top-level section `## Tools`.

Reply:
done.